packages/validator-ajv8/test/validator.test.ts

```diff
@@ -394,13 +396,925 @@ describe("AJV8Validator", () => {
       });
     });
   });
+  describe("default options, with Ajv2019", () => {
```

Comment:
@heath-freenome 
I'm assuming this is all copy/paste with the new class? Let me know if I should take a closer look at this.

Reply:
There was one notable change that I commented on, but yeah